core/test/gather/gatherers/trace-elements-test.js

```diff
@@ -104,167 +98,15 @@ describe('Trace Elements gatherer - GetTopLayoutShiftElements', () => {
-          node_id: 2,
-          old_rect: [0, 0, 200, 100],
-        },
-      ], true)
-    );
-    const processedTrace = await ProcessedTrace.request(trace, {computedCache: new Map()});
-
-    const result = TraceElementsGatherer.getTopLayoutShiftElements(processedTrace);
-    expect(result).toEqual([
-      {nodeId: 1, score: 1},
-      {nodeId: 2, score: 1},
-    ]);
-  });
-
-  it('does ignore later trace events with input', async () => {
-    const trace = createTestTrace({});
-    trace.traceEvents.push(
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 1,
-          old_rect: [0, 0, 200, 100],
-        },
-      ]),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 2,
-          old_rect: [0, 0, 200, 100],
-        },
-      ], true)
-    );
-    const processedTrace = await ProcessedTrace.request(trace, {computedCache: new Map()});
-
-    const result = TraceElementsGatherer.getTopLayoutShiftElements(processedTrace);
-    expect(result).toEqual([
-      {nodeId: 1, score: 1},
-    ]);
-  });
-
-  it('correctly ignores trace events with input (complex)', async () => {
-    const trace = createTestTrace({});
-    trace.traceEvents.push(
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 1,
-          old_rect: [0, 0, 200, 100],
-        },
-      ], true),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 2,
-          old_rect: [0, 0, 200, 100],
-        },
-      ], true),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 3,
-          old_rect: [0, 0, 200, 100],
-        },
-      ]),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 4,
-          old_rect: [0, 0, 200, 100],
-        },
-      ]),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 5,
-          old_rect: [0, 0, 200, 100],
-        },
-      ], true),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 6,
-          old_rect: [0, 0, 200, 100],
-        },
-      ], true),
-      makeLayoutShiftTraceEvent(1, [
-        {
-          new_rect: [0, 0, 200, 200],
-          node_id: 7,
-          old_rect: [0, 0, 200, 100],
-        },
-      ])
-    );
-    const processedTrace = await ProcessedTrace.request(trace, {computedCache: new Map()});
-
-    const result = TraceElementsGatherer.getTopLayoutShiftElements(processedTrace);
-    expect(result).toEqual([
-      {nodeId: 1, score: 1},
-      {nodeId: 2, score: 1},
-      {nodeId: 3, score: 1},
-      {nodeId: 4, score: 1},
-      {nodeId: 7, score: 1},
-    ]);
-  });
-
-  it('combines scores for the same nodeId accross multiple shift events', async () => {
```

Comment:
This test case was moved to `computed/metrics/cumulative-layout-shift-test.js`. The remaining tests were already covered in that file.